domain/servicefactory/model.go

```diff
@@ -18,12 +19,13 @@ type ModelFactory struct {
 // Config returns the model's configuration service. A ModelDefaultsProvider
 // needs to be supplied for the model config service. The provider can be
 // obtained from the controller service factory model defaults service.
-func (f *ModelFactory) Config(
+func (s *ModelFactory) Config(
```

Comment:
Why did the receiver name change?

Reply:
Was just trying to align with the controller one.